.gitignore

```diff
@@ -104,3 +104,6 @@ dist
 .tern-port
 
 server/node_modules/
+
+.env
+local.db
```

Comment:
just named my database local.db but we can probably do *.db

Reply:
let's do *.db so we can name our database whatever we want locally

Reply:
did it